### PR TITLE
[Config Change] refactor: adapt config change plugins to unified config model

### PR DESCRIPTION
**Pull request by Zhengcy05**
## Background

Part of alibaba/nacos#15617.

This PR handles the Config Change plugin family migration for Nacos 3.3 unified plugin management.

## Changes

- Adapt `webhook` to `PluginConfigSpec`
  - declare `webhookUrl`
  - declare `contentMaxCapacity`
  - keep legacy aliases for old static keys
  - apply and expose an effective config snapshot

- Adapt `whitelist` to `PluginConfigSpec`
  - introduce canonical item key `suffixes`
  - keep legacy `suffixs` as an alias
  - apply and expose an effective config snapshot

- Keep `fileformatcheck` zero-config as requested by the issue checklist.

- Add focused unit tests for:
  - config definitions
  - aliases
  - runtime effect mode
  - defensive current config snapshots
  - whitelist filtering with canonical and legacy config keys

## Validation

- `mvn -pl plugin/config -am -DskipTests install`
- `mvn -pl nacos-config-change-plugin-ext/nacos-webhook-config-change-plugin,nacos-config-change-plugin-ext/nacos-whitelist-config-change-plugin,nacos-config-change-plugin-ext/nacos-fileformat-config-change-plugin -am test`
- `git diff --check`

### CODE REVIEW

File: nacos-config-change-plugin-ext/nacos-webhook-config-change-plugin/src/main/java/com/alibaba/nacos/plugin/config/WebHookConfigChangePluginService.java [hunk 1]
```diff
@@ -47,53 +54,97 @@
  * @author liyunfei
  **/
 public class WebHookConfigChangePluginService implements ConfigChangePluginService {
-    
+
```
Comment (KomachiSion) [inline nacos-config-change-plugin-ext/nacos-webhook-config-change-plugin/src/main/java/com/alibaba/nacos/plugin/config/WebHookConfigChangePluginService.java]: don't change the indent.

please check for all classes all lines in PR.

File: nacos-config-change-plugin-ext/nacos-webhook-config-change-plugin/src/main/java/com/alibaba/nacos/plugin/config/WebHookConfigChangePluginService.java [hunk 2]
```diff
@@ -47,53 +54,97 @@
...
     private static final Logger LOGGER = LoggerFactory.getLogger(WebHookConfigChangePluginService.class);
-    
+
+    private static final String WEBHOOK_URL = "webhookUrl";
+
+    private static final String CONTENT_MAX_CAPACITY = "contentMaxCapacity";
```
Comment (KomachiSion) [inline nacos-config-change-plugin-ext/nacos-webhook-config-change-plugin/src/main/java/com/alibaba/nacos/plugin/config/WebHookConfigChangePluginService.java]: `contentMaxCapacity` is declared as `ConfigItemType.NUMBER`, but the unified basic checker accepts any valid `BigDecimal`. Values such as `1.5`, `-1`, or a number exceeding the `int` range can therefore pass the plugin configuration API and `applyConfig()`, but later fail in `Integer.parseInt()` or `substring()` during webhook execution.

Please validate and parse this value when applying the configuration, require a positive integer within the supported range, and add tests for decimal, non-positive, and overflow values. Invalid runtime configuration should be rejected by the update operation instead of failing later in the asynchronous webhook path.

It would also be better to use canonical kebab-case item keys such as `webhook-url` and `content-max-capacity`, while retaining the existing camel-case names and historical full keys as aliases.

**Review by KomachiSion — CHANGES_REQUESTED** [review]
The overall ConfigSpec integration direction looks correct, and the whitelist alias handling is compatible with the unified resolver. The webhook numeric configuration still needs stricter validation before this PR is ready.

File: nacos-config-change-plugin-ext/nacos-webhook-config-change-plugin/src/main/java/com/alibaba/nacos/plugin/config/WebHookConfigChangePluginService.java (hunk 1 above)
Comment (Zhengcy05) [reply nacos-config-change-plugin-ext/nacos-webhook-config-change-plugin/src/main/java/com/alibaba/nacos/plugin/config/WebHookConfigChangePluginService.java]: Done.

File: nacos-config-change-plugin-ext/nacos-webhook-config-change-plugin/src/main/java/com/alibaba/nacos/plugin/config/WebHookConfigChangePluginService.java (hunk 2 above)
Comment (Zhengcy05) [reply nacos-config-change-plugin-ext/nacos-webhook-config-change-plugin/src/main/java/com/alibaba/nacos/plugin/config/WebHookConfigChangePluginService.java]: Done.

**Review by KomachiSion — APPROVED** [review]
